engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -117,48 +115,14 @@ class WorkflowManagerActor(params: WorkflowManagerActorParams)
   private val tag = self.path.name
 
   private var abortingWorkflowToReplyTo = Map.empty[WorkflowId, ActorRef]
-  private var shutdownHookThreadOption: Option[ShutdownHookThread] = None
+  private var nextPollCancellable: Option[Cancellable] = None
+  private var preventNewWorkflowsFromStarting: Boolean = false
```

Comment:
Yes and no, it's very slightly modifying the behavior of the "Running" state by deciding whether or not a method should be called, which could be said to be a different state I agree. I can try to play with state functions to try and make it so, what I wanted to avoid was copy/pasting the whole "Running" state function to a new "Running but not polling" one with just 2 lines being removed.